src/query/service/src/servers/http/v1/query/http_query.rs

```diff
@@ -141,6 +143,12 @@ impl HttpSessionConf {
     }
 }
 
+#[derive(Deserialize, Debug, Clone)]
+pub struct StageAttachmentConf {
+    pub(crate) location: String,
+    pub(crate) params: Option<BTreeMap<String, String>>,
```

Comment:
is this `params` only for format_options?
is one location enough?
which form of location do we intend to use? name? internal/external/uri?

Reply:
> is this params only for format_options?
for both format options and copy options.

> is one location enough?
> which form of location do we intend to use? name? internal/external/uri?

yes, we use format like `@stage_name/path`

Reply:
> yes, we use format like @stage_name/path
ok 

> for both format options and copy options.

I prefer to separate them and reuse code in parser

Reply:
> yes, we use format like @stage_name/path

we can add a little comments here.